test_pull_requests

```diff
@@ -803,6 +843,7 @@ popd
     end
 
     # Updates all the bot markers for the involved pull requests.  To be called after you have validated the tests should be kicked off again
+    # update_evaluated_markers 
```

Comment:
To be clear what I meant.  The rest of the method comments start with the method name and then describe what they do.  This one is the opposite.